src/uct/tcp/tcp_ep.c

```diff
@@ -920,10 +929,13 @@ unsigned uct_tcp_ep_progress_am_rx(uct_tcp_ep_t *ep)
         } else {
             ucs_assert(hdr->am_id == UCT_TCP_EP_CM_AM_ID);
             handled += 1 + uct_tcp_cm_handle_conn_pkt(&ep, hdr + 1, hdr->length);
+            /* coverity[check_after_deref] */
```

Comment:
btw why is coverity failing?

Reply:
> btw why is coverity failing?

Coverity thinks that the pointer to `ep` can be `NULL` above, since we have
```C
if (ep == NULL) ...
```
I tried adding `ucs_assert(ep != NULL)`, but it didn't help.
I don't know a better way to explain Coverity that this is wrong and can't happen